### PR TITLE
Front-end Unit Tests: Dashboard Controller

### PR DESCRIPTION
**Pull request by Sanji515**
**PR contains the front-end unit tests of** `Dashboard Controller`

### CODE REVIEW

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+            isUserDetails = null;
+            isPresentChallenge = null;
+            isHostTeam = true;
+            isParticipantTeam = null;
+            successResponse = [
+                {
+                    id: 1,
+                    team_name: "Host team name 1",
+                    created_by: "user 1",
+                    team_url: "Team url 1"
```
Comment (RishabhJain2018) [inline frontend/tests/controllers-test/dashCtrl.test.js]: Please add a valid URL link here. something say `https://team1.url`

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+                },
+                {
+                    id: 2,
+                    team_name: "Host team name 2",
+                    created_by: "user 2",
+                    team_url: "Team url 2"
```
Comment (RishabhJain2018) [inline frontend/tests/controllers-test/dashCtrl.test.js]: Same as above

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+            isUserDetails = null;
+            isPresentChallenge = null;
+            isHostTeam = null;
+            isParticipantTeam = true;
+            successResponse = [
+                {
+                    id: 1,
+                    team_name: "Participants team name 1",
+                    created_by: "user 1",
+                    team_url: "Team url 1"
```
Comment (RishabhJain2018) [inline frontend/tests/controllers-test/dashCtrl.test.js]: Please add a valid URL link here. something say `https://team1.url`

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+                },
+                {
+                    id: 2,
+                    team_name: "Participants team name 2",
+                    created_by: "user 2",
+                    team_url: "Team url 2"
```
Comment (RishabhJain2018) [inline frontend/tests/controllers-test/dashCtrl.test.js]: Please add a valid URL link here. something say `https://team1.url`

File: frontend/tests/controllers-test/dashCtrl.test.js [hunk 5]
```diff
@@ -0,0 +1,282 @@
+'use strict';
+
+describe('Unit tests for Dashboard Controller', function () {
```
Comment (RishabhJain2018) [inline frontend/tests/controllers-test/dashCtrl.test.js]: `Unit tests for Dashboard Controller` -> `Unit tests for dashboard controller`

File: frontend/tests/controllers-test/dashCtrl.test.js (hunk 1 above)
Comment (Sanji515) [reply frontend/tests/controllers-test/dashCtrl.test.js]: @RishabhJain2018 done

File: frontend/tests/controllers-test/dashCtrl.test.js (hunk 2 above)
Comment (Sanji515) [reply frontend/tests/controllers-test/dashCtrl.test.js]: done

File: frontend/tests/controllers-test/dashCtrl.test.js (hunk 3 above)
Comment (Sanji515) [reply frontend/tests/controllers-test/dashCtrl.test.js]: done

File: frontend/tests/controllers-test/dashCtrl.test.js (hunk 4 above)
Comment (Sanji515) [reply frontend/tests/controllers-test/dashCtrl.test.js]: done

File: frontend/tests/controllers-test/dashCtrl.test.js (hunk 5 above)
Comment (Sanji515) [reply frontend/tests/controllers-test/dashCtrl.test.js]: done

**Pull request merged**